scRNA-seq/data/hodgkins/hs_mitochondrial_genes.tsv

```diff
@@ -0,0 +1,38 @@
+gene_id	gene_name	seqnames	start	end	width	strand	gene_biotype	seq_coord_system	description	gene_id_version	symbol	entrezid
```

Comment:
Wanted to just check in about the decision to include this in version control and how that might impact `skel` prep.

Reply:
I don't think it will affect skel prep, but yeah, I don't know. I figured it was pretty small and the mouse file was already in there, but I felt _bad_ when I added it.

Reply:
A better question I could have asked instead: Should this be in S3 instead of tracked here?

Reply:
Since the mouse version file just came up as an issue for exercise notebooks, the answer to this is yes. I filed a separate issue: https://github.com/AlexsLemonade/training-modules/issues/472 to update the scripts (and remove this version).